app/controllers/spree/admin/back_in_stock_notifications_controller.rb

```diff
@@ -8,6 +8,8 @@ class BackInStockNotificationsController < ResourceController
       def index
         @back_in_stock_notifications = @back_in_stock_notifications
           .pending
+          .includes(variant: :product)
+          .includes(:stock_location)
```

Comment:
You can do this in one call if you prefer:

```ruby
          .includes(:stock_location, variant: :product)
```

Reply:
Thanks. I had tried it the other way around but realise now why it didn't work.